README.md

```diff
@@ -1,3 +1,3 @@
-**http://todotxt.org** [![](https://img.shields.io/gitter/room/todotxt/Lobby.svg)](https://gitter.im/todotxt/Lobby)
+**https://todotxt.org** [![](https://img.shields.io/gitter/room/todotxt/Lobby.svg)](https://gitter.im/todotxt/Lobby)
```

Comment:
This change cannot be made yet — see #32.

Reply:
When I open the https-version of the page it's already available online, but the connection ist not secure. De facto the same as http, isn't it? So we can make the change and wait for ginatrapani to update the DNS entry.

But if not I will remove this commit, so we can merge the other ones.

Reply:
Don't you get a browser warning if you visit the https link? I don't think that would be an equivalent experience for the user, compared to visiting an http page and getting there without issues — even though technically the site is indeed equally unsecured in both.

Reply:
Yes I do. Got your point. So I'll remove that.